migrations/20200629223354_create_tx_out_address_index.js

```diff
@@ -0,0 +1,7 @@
+exports.up = async (knex) => knex.schema.table('tx_out', (table) => {
+  table.index('address', 'idx_tx_out_address', 'hash')
```

Comment:
hmm in hindsight, I would prefix the index with "adalite" to mitigate the risk of clashing with future migrations of cardano-sb-sync